src/pages/_app.tsx

```diff
@@ -1,16 +1,57 @@
 // import '@/styles/globals.css'
 import type { AppProps } from "next/app";
 import Head from "next/head";
-import { DSProvider } from "@nypl/design-system-react-components";
+import {
+  DSProvider,
+  useFeedbackBox,
+} from "@nypl/design-system-react-components";
+import React from "react";
 
 export default function App({ Component, pageProps }: AppProps) {
+  const [view, setView] = React.useState("form");
+  const { onOpen, isOpen, onClose, FeedbackBox } = useFeedbackBox();
+  const onSubmit = async (values) => {
+    const response = await fetch("/api/feedback", {
```

Comment:
Do either async/await or the then/catch syntax but not both. Either is fine but the async/await syntax has become more conventional (callback functions can be annoying).

With async/await, wrap this in a try/catch statement:
```
    try {
      const response = await fetch("/api/feedback", {
        method: "POST",
        headers: {
          Accept: "application/json",
          "Content-Type": "application/json",
        },
        body: JSON.stringify(values),
      });

      console.log("response is: ", response);

      if (response.ok) {
        // ...
        setView("confirmation");
      } else {
        setView("error");
      }
    } catch (error) {
      // Reject the promise according to your application.
      // And then call:
      console.log("error is: ", error);
      setView("error");
    }
```